it-follows/game.lua

```diff
@@ -56,6 +58,7 @@ local function game_over()
   local t, e = s:lines()
   state.game_over = { time = t, enemies = e }
   state.status = "game_over"
+  audio:bgmUpdate(state.status)
```

Comment:
Here I'd do

```suggestion
  audio:stop()
```

Reply:
At like 85, we know if the status is `started`, so we can add a code that checks if the audio is playing. If not, we play it

```lua
if audio.status == 'playing' then
  audio:play()
end
```